frontend/src/metabase-lib/queries/StructuredQuery.ts

```diff
@@ -640,7 +643,27 @@ class StructuredQueryInner extends AtomicQuery {
    * @returns an array of aggregation options for the currently selected table
    */
   aggregationOperators(): AggregationOperator[] {
-    return (this.table() && this.table().aggregationOperators()) || [];
+    const expressionFields = this.expressionDimensions().map(
+      expressionDimension => expressionDimension.field(),
+    );
+
+    const table = this.table();
+    return (
+      (table &&
+        getAggregationOperators(table, [
+          ...expressionFields,
+          ...table.fields,
+        ])) ||
+      []
+    );
+  }
+
+  aggregationOperatorsLookup(): Record<string, AggregationOperator> {
+    return createLookupByProperty(this.aggregationOperators(), "short");
+  }
+
+  aggregationOperator(short: string): AggregationOperator {
```

Comment:
`aggregationOpertors` (with an ending `s`) was introduced in https://github.com/metabase/metabase/pull/11032 in `Table`, `StructuredQuery` and a number of other Metabase lib inteties it also introduced `aggregationOperator` but only to `Dimension`. 

This function `aggregationOperator` (_without_ an ending `s`) was only additionally introduced in `Table` and `Field` in https://github.com/metabase/metabase/pull/12859

This change brings it closer to how `Table`'s `aggregationOperators` and `aggregationOperator` were implemented.